lib/beaker/host_prebuilt_steps.rb

```diff
@@ -107,6 +107,7 @@ def validate_host host, opts
         when host['platform'] =~ /cumulus/
           check_and_install_packages_if_needed(host, CUMULUS_PACKAGES)
         when (host['platform'] =~ /windows/ and host.is_cygwin?)
```

Comment:
Updated for a more verbose comment for `cygwin_installed?` and to point out that `is_cygwin?` is just a type check for `Windows::Host`.